pkg/volume/volume.go

```diff
@@ -65,6 +66,12 @@ type Recycler interface {
 	Recycle() error
 }
 
+// Create adds a new resource in the storage provider and creates a PersistentVolume for the new resource.
+// Calls to Create should block until complete.
+type Creater interface {
+	Create() (*api.PersistentVolume, error)
```

Comment:
@thockin 

> The NetApp NFS provisioner is a totally different beast than the GCE
provisioner.    I do not think we should have a single provisioner
that knows how to provision every volume type.

I wasn't talking about a single provisioner that knows how to provision all types.  I was saying, we'll have an EFS provisioner and netapp provisioner, do we need to differentiate entire plugins for those since they're both NFS on the client side (afaik), or do we need to have a mapping of the PV concerns to plugins that allows for us to DNR on volume plugins just to differentiate provisioners.